.github/workflows/build.yml

```diff
@@ -0,0 +1,88 @@
+name: Build
+
+on:
+    push:
+        branches: [master]
+    pull_request:
+    release:
+        types: [created]
+
+jobs:
+    tests:
+        runs-on: ubuntu-latest
+        name: Build and test
+        strategy:
+            matrix:
+                php: ["7.2", "7.4"]
```

Comment:
Could you bump it to 7.3? I think it's time to drop 7.2; Maybe we can include 8.0 as well